incubator/java-openliberty/image/project/pom.xml

```diff
@@ -155,7 +155,71 @@
                             <artifactId>liberty-maven-plugin</artifactId>
                             <version>${version.liberty-maven-plugin}</version>
                             <configuration>
-                                <installDirectory>${openliberty.install.dir}/wlp</installDirectory>
+                                <installDirectory>/opt/ol/wlp</installDirectory>
```

Comment:
Should we be using the wlp.install.dir property defined above?

Reply:
This threw me...  the most reuse would be gained by defining a prop pointing to `/opt/ol` I was hesitant to coin a new name... so I just left it expanded.  

Besides configuring the install, the "wlp.install.dir" is used by the user to copy the project dependencies, for test, and appsody build.